.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Pull request
+about: Tell us about your contribution
+---
+
+<!-- Thanks for contributing to Cross-cluster-connectivity! -->
```

Comment:
nit: I think we've esablished the name as "Cross-cluster Connectivity" per the opening line of the README. Let's use that for consistency (and do a find/replace all later if we update our capitalization/dasherization preferences)

Reply:
Good catch, that was unintentional.